docs/lib-documentation.md

```diff
@@ -246,17 +244,13 @@ _Found a bug? File it on [the repo](https://github.com/kurtosis-tech/kurtosis-cl
 <!-- TODO Make the function definition not include args or return values, so we don't get these huge ugly links that break if we change the function signature -->
 <!-- TODO make the reference names a) be properly-cased (e.g. "Service.isAvailable" rather than "service_isavailable") and b) have an underscore in front of them, so they're easy to find-replace without accidentally over-replacing -->
 
-[availabilitychecker]: #availabilitychecker
```

Comment:
Just to double-check - for all the links being removed, have you searched throughout the document to ensure that they're not used (so we don't end up with broken links)?

Reply:
No, I didn't search for that, I'll do it.

Reply:
Now I did it